### DISCUSSION TITLE
web-vault folder gone when upgrading to 1.28.0

### DISCUSSION THREAD

**Discussion by wein-geist** [dani-garcia/vaultwarden#3471] [Q&A]
Hey, 

I run Vaultwarden in a TrueNAS iocage jail. Recently, I finally upgraded to TrueNAS-13.0-U4  (FreeBSD 13.1 RELEASE-p7). 

When trying to pkg upgrade, it suggests me version 1.28.0
        vaultwarden: 1.27.0_4 -> 1.28.0

However, after running all upgrades, the web-vault folder under /usr/local/www/vaultwarden is gone. Of course when trying to start Vaultwarden, it complains that it cant find the web-vault folder. 

It is reproduceable, it happened twice. Any idea on what went wrong? 

Thanks in advance, 
wein-geist

**Comment by stefan0xC** [comment]
Seems like the maintainers of this [third-party package](https://github.com/dani-garcia/vaultwarden/wiki/Third-party-packages) [split the web-vault into a separate package](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=270182).
